spec/rspec/core/example_group_spec.rb

```diff
@@ -56,12 +56,12 @@ def metadata_hash(*args)
     end
 
     it 'does not treat the first argument as a metadata key even if it is a symbol' do
-      group = RSpec.describe(:symbol)
+      group = with_isolated_stderr { RSpec.describe(:symbol) }
       expect(group.metadata).not_to include(:symbol)
     end
 
     it 'treats the first argument as part of the description when it is a symbol' do
-      group = RSpec.describe(:symbol)
+      group = with_isolated_stderr { RSpec.describe(:symbol) }
```

Comment:
These should be removed/replaced really?

Reply:
I'm on the fence with this, too.
On the one hand, the whole thing is to warn users when they think they add metadata (e.g. `RSpec.describe :focus`) which is treated as a docstring.
On the other hand, we're just issuing a warning, not hard failing, and we need some specs to prevent regressions in case we decide to rearrange things related to `described_class` or metadata extraction.

Reply:
Yeah fair, maybe add an alias for "we expect a warning"?

Reply:
Done